students/saleta_sebastian/lesson_05_lists/character_picture_grid.py

```diff
@@ -0,0 +1,26 @@
+def get_picture_grid(raw_grid):
```

Comment:
I'm asking because I run your solution with Python 2.7 where it works and when I run with Python 3.5 they do n't work. The problem is that the value returned by built-in zip function is a bit different in both versions 😃